ui/src/Citation.tsx

```diff
@@ -45,6 +46,11 @@ export class Citation extends React.PureComponent<CitationProperties> {
                     </span>
                   </div>
                 ) : null}
+                {paper.primaryPaperLink !== null ? (
```

Comment:
Will this ever be null? If so, can you update the `Paper` interface declaration for the `primaryPaperLink` property in `api/src/s2-api.ts` to indicate that it's sometimes null?